multiqc/modules/fastqc/fastqc.py

```diff
@@ -312,9 +312,12 @@ def read_count_plot(self):
             pd = self.fastqc_data[s_name]["basic_statistics"]
             pdata[s_name] = dict()
             try:
-                pdata[s_name]["Duplicate Reads"] = int(
-                    ((100.0 - float(pd["total_deduplicated_percentage"])) / 100.0) * pd["Total Sequences"]
-                )
+                if math.isnan(float(pd["total_deduplicated_percentage"])): 
+                    pdata[s_name]["Duplicate Reads"] = math.nan
```

Comment:
This can break JSON encoding and other stuff. Can we just unset, or set to `"-"` or something instead please?

Reply:
Thanks for the suggestion. Is the new patch looks ok?